pkg/sql/crdb_internal.go

```diff
@@ -8034,3 +8035,48 @@ CREATE TABLE crdb_internal.node_tenant_capabilities_cache (
 		return nil
 	},
 }
+
+func populateFromQuery(query string, op string) populateFunc {
+	return func(ctx context.Context, p *planner, _ catalog.DatabaseDescriptor, addRow func(...tree.Datum) error) (retErr error) {
+		rows, err := p.ExecCfg().InternalDB.Executor().QueryIteratorEx(
+			ctx,
+			op,
+			nil,
+			sessiondata.RootUserSessionDataOverride,
+			query,
+		)
+		if err != nil {
+			return err
+		}
+		defer func() {
+			retErr = errors.CombineErrors(retErr, rows.Close())
+		}()
+		for {
+			ok, err := rows.Next(ctx)
+			if err != nil {
+				return err
+			}
+			if !ok {
+				break
+			}
+			err = addRow(rows.Cur()...)
+			if err != nil {
+				return err
+			}
+		}
+		return nil
+	}
+}
+
+var crdbInternalSystemPrivileges = virtualSchemaTable{
+	comment: `vtable backing SHOW SYSTEM GRANTS`,
+	schema: `
+CREATE TABLE crdb_internal.system_privileges (
```

Comment:
Why do user privileges get a `kv_` prefix?

Reply:
According to https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/crdb_internal.go#L107 this vtable needs the kv_ prefix because it's populated (ultimately) by performing a KV scan on a physical table.